### PR TITLE
adding doc on custom certs with incert

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This PR adds a new doc on inserting custom certificates into an Image with `incert`. It's based off off [Adrian's video on the same topic](https://drive.google.com/file/d/1Wl8XBhijRoHQ73nu8S7MDoN5y1oz4P-o/view)

### What should this PR do?

Resolves https://github.com/chainguard-dev/customer-issues/issues/1200 

### Why are we making this change?

This helps customers whose orgs have their own custom certificate authorities. 

### What are the acceptance criteria? 

Should function correctly, have correct spelling and grammar, and should make sense from a logical and technical perspective.

### How should this PR be tested?

Tech test + close read through, please!

### CODE REVIEW

File: content/chainguard/chainguard-images/working-with-images/incert-custom-certs.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+```
+```
+csr.json  selfsigned.csr  selfsigned-key.pem  selfsigned.pem
+```
+
+With these files in place you can move on to creating an Nginx container that uses these certificates to provide TLS.
+
+## Create an Nginx container that uses self-signed certificates for TLS
+
+Now that you've created the certificate infrastructure, you can create an Nginx container that uses them to provide TLS. Later on, we will attempt to reach this Nginx container with a `curl` container we built using `incert`, testing whether it can successfully insert the `selfsigned.pem` certificate into it.
```
Comment (amouat) [inline content/chainguard/chainguard-images/working-with-images/incert-custom-certs.md:109]: Maybe "testing that incert correctly installed the `selfsigned.pem` certificate into it"?

File: content/chainguard/chainguard-images/working-with-images/incert-custom-certs.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+`incert` is a Go program from Chainguard that appends CA certificates to Docker images and pushes the modified image to a specified registry. This tool is still in active development, so feedback is welcome.
+
+Run the following command to build a new image using Chainguard's `curl` image as its base and insert the `selfsigned.pem` certificate into it. 
+
+```sh
+incert -ca-certs-file selfsigned.pem -platform linux/arm64 -image-url cgr.dev/chainguard/curl:latest -dest-image-url ttl.sh/curl-with-cert:1h
+```
+
+This command uses the `-ca-certs-file` option to specify that `incert` should use the `selfisgned.pem` certificate file; the `-platform` option to specify that it wants to build an image for `linux/arm64`; the `-image-url` option to specify the image we want to build on as our base image (here we specify Chainguard's `curl` image); and the `-dest-image-url` to pass the registry where we want the resulting image to be uploaded to.
```
Comment (amouat) [inline content/chainguard/chainguard-images/working-with-images/incert-custom-certs.md:219]: I guess it's worth mentioning they should change the -platform argument to match the host platform.

**Review by amouat — APPROVED** [review]
This looks good to me! A couple of minor points.

It's probably worth checking that we've capitalised nginx in the same way as our other tutorials (I note that nginx.com and nginx.org seem to disagree here, I would go with nginx.org).

I think it makes sense to link the video once released.

File: content/chainguard/chainguard-images/working-with-images/incert-custom-certs.md (hunk 2 above)
Comment (SharpRake) [reply content/chainguard/chainguard-images/working-with-images/incert-custom-certs.md:219]: good call!

**Pull request merged**